static/src/js/components/AccessMethod/AccessMethod.jsx

```diff
@@ -40,6 +23,7 @@ import RadioList from '../FormFields/RadioList/RadioList'
 import Spinner from '../Spinner/Spinner'
 
 import './AccessMethod.scss'
+import SwodlrForm from './SwodlrForm'
```

Comment:
Move this import to other component imports, we always have the scss file imported last